### DISCUSSION TITLE
Compatible with EA firmware 3.0.10?

### DISCUSSION THREAD

**Discussion by Calimerorulez** [fabianishere/udm-iptv#193] [Q&A]
Today, UI released firmwares 3.0.10 for the UDM-SE and UDR. Can anyone confirm if udm-iptv is compatible with these firmwares, before I try to upgrade? :)

**Comment by Calimerorulez** [comment]
OK, I took the plunge. Installing gave me the following error (using KPN as profile):

`/tmp/udm-iptv.config.EPWYG7: 134: .: cannot open /usr/lib/udm-iptv/profiles/kpn.profile: No such file`

I have to test the TV though.

**Reply by rsterrenburg** [reply]
I had to reinstall udm-iptv after upgrading to 3.0.10 on UDM-SE, but then it worked (including TV). Provider KPN NL.
